acceptance/lib/acceptance/install_ruby.rb

```diff
@@ -0,0 +1,6 @@
+def install_ruby_from_source(host, url = "https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.4.tar.gz")
```

Comment:
We should mirror this instead of downloading from ruby-lang every time.

Reply:
Which of course we didn't do for Windows.

Reply:
Eh, still a good idea :P